src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -29,7 +29,7 @@ var SELECTION_RESTORATION = {
    */
   initialize: ReactInputSelection.getSelectionInformation,
   /**
-   * @param {Selection} sel Selection information returned from `initialize`.
+   * @param {Selection} Selection information returned from `initialize`.
```

Comment:
The code was already correct as-is.  The `sel` is referring to the variable name that is a parameter to the function.  You can see this pattern throughout the inline docs: https://github.com/facebook/react/blob/3b96650e39ddda5ba49245713ef16dbc52d25e9e/src/renderers/dom/client/ReactMount.js#L88-L91